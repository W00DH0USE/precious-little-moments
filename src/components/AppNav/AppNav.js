import React from 'react';
import { Link } from 'react-router-dom';

import './AppNav.css';

// Displayed while user is logged in and on the profilePage
function AppNav(props) {
  return (
    <nav className="Header" style={{ borderBottom: "1px solid #eee" }}>
        <h1>
          <Link to="/">
            PLM
          </Link>
        </h1>

        <div className="SignOut">
          <div className='Header__logged-in'>
            <Link
              to='/profile'>
              Profile
            </Link>
            <Link
              to='/addPost'>
              Create Moment
            </Link>
            <Link
              onClick={props.handleLogout}
              to='/'>
              Logout
            </Link>
          </div>
        </div>
      </nav>
  )
}

export default AppNav;