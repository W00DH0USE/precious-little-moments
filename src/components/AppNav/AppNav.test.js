import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AppNav from './AppNav';


describe(`AppNav component`, () => {
    it('renders without crashing', () => {
        const wrapper = shallow(<AppNav />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});