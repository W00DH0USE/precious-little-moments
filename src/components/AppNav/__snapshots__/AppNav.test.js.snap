// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppNav component renders without crashing 1`] = `
<Fragment>
  <Menu
    bodyClassName=""
    burgerBarClassName=""
    burgerButtonClassName=""
    className=""
    crossButtonClassName=""
    crossClassName=""
    disableAutoFocus={false}
    disableCloseOnEsc={false}
    htmlClassName=""
    id=""
    itemClassName=""
    itemListClassName=""
    menuClassName=""
    morphShapeClassName=""
    noOverlay={false}
    onStateChange={[Function]}
    outerContainerId=""
    overlayClassName=""
    pageWrapId=""
    role="navigation"
    styles={Object {}}
    width={300}
  >
    <Link
      className="menu-item"
      to="/posts"
    >
      Home
    </Link>
    <Link
      className="menu-item"
      to="/addPost"
    >
      Create a Post
    </Link>
    <Link
      className="menu-item"
      to="/profile"
    >
      Profile
    </Link>
    <Link
      to="/"
    >
      Log Out
    </Link>
  </Menu>
</Fragment>
`;
